### PR TITLE
Minor Team API changes

### PR DESCRIPTION
**Pull request by adityatMS**

### CODE REVIEW

File: Include/xsapi/tournaments.h [hunk 1]
```diff
@@ -261,7 +261,7 @@ class previous_match_metadata
     /// <summary>
     /// The timestamp when the match ended.
     /// </summary>
-    _XSAPIIMP const utility::datetime& match_end_time() const;
+    _XSAPIIMP const utility::datetime& end_time() const;
```
Comment (edjamesmsft) [inline Include/xsapi/tournaments.h]: >end_time [](start = 39, length = 8)

is there a reason we don't have start_time here as well?

File: Include/xsapi/tournaments.h [hunk 2]
```diff
@@ -433,6 +439,43 @@ class team_info
 };
 
 /// <summary>
+/// Represents a summary of team details for the user participating in the tournament.
+/// </summary>
+class team_summary
```
Comment (edjamesmsft) [inline Include/xsapi/tournaments.h]: >team_summary [](start = 6, length = 12)

I like this name :)

File: Tests/UnitTests/Tests/Services/TournamentsTests.cpp [hunk 3]
```diff
@@ -80,6 +80,7 @@ DEFINE_TEST_CLASS(TournamentsTests)
         VERIFY_ARE_EQUAL(teamInfo->Name->Data(), teamToVerify[L"name"].as_string());
         VERIFY_ARE_EQUAL(teamInfo->Standing->Data(), teamToVerify[L"standing"].as_string());
         VERIFY_ARE_EQUAL_INT(teamInfo->Ranking, teamToVerify[L"ranking"].as_integer());
+        //VERIFY_ARE_EQUAL(teamInfo->ContinuationUri->Data(), teamToVerify[L"name"].as_string());
```
Comment (edjamesmsft) [inline Tests/UnitTests/Tests/Services/TournamentsTests.cpp]: Remove?

File: Include/xsapi/tournaments.h (hunk 1 above)
Comment (adityatMS) [inline Include/xsapi/tournaments.h]: start_time is part of the match_metadata class which is common to both previous & current match metadata

---
In reply to: [107030620](https://github.com/Microsoft/xbox-live-api/pull/106#discussion_r107030620) [](ancestors = 107030620)

File: Tests/UnitTests/Tests/Services/TournamentsTests.cpp (hunk 3 above)
Comment (adityatMS) [inline Tests/UnitTests/Tests/Services/TournamentsTests.cpp]: I should ideally fix this. Thanks for pointing it out.

---
In reply to: [107031201](https://github.com/Microsoft/xbox-live-api/pull/106#discussion_r107031201) [](ancestors = 107031201)

File: Include/xsapi/tournaments.h (hunk 2 above)
Comment (adityatMS) [inline Include/xsapi/tournaments.h]: Thanks :)

---
In reply to: [107030711](https://github.com/Microsoft/xbox-live-api/pull/106#discussion_r107030711) [](ancestors = 107030711)

**Pull request merged**